testData/solutions/linux.json

```diff
@@ -13,7 +13,8 @@
             {
                 "type": "gpii.gsettings",
                 "options": {
-                    "schema": "org.gnome.desktop.a11y.magnifier"
+                    "schema": "org.gnome.desktop.a11y.magnifier",
+                    "verifySettings": true
```

Comment:
I don't think this is required for any linux settings - was there any evidence that the gsettings system did not act synchronously?